File: sdkwork-ai-sdk-go/types/batch_response.go
package types

// 批量任务响应
type BatchResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Endpoint string `json:"endpoint"`
	Errors Errors `json:"errors"`
	Status string `json:"status"`
	Metadata map[string]string `json:"metadata"`
	InputFileId string `json:"input_file_id"`
	OutputFileId string `json:"output_file_id"`
	ErrorFileId string `json:"error_file_id"`
	CreatedAt int `json:"created_at"`
	CompletionWindow string `json:"completion_window"`
	ExpiresAt int `json:"expires_at"`
	StartedAt int `json:"started_at"`
	CompletedAt int `json:"completed_at"`
	FailedAt int `json:"failed_at"`
	ExpiredAt int `json:"expired_at"`
	CancelledAt int `json:"cancelled_at"`
	RequestCounts RequestCounts `json:"request_counts"`
}

File: sdkwork-ai-sdk-csharp/Models/BatchResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class BatchResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Endpoint { get; set; }
        public Errors? Errors { get; set; }
        public string? Status { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public string? InputFileId { get; set; }
        public string? OutputFileId { get; set; }
        public string? ErrorFileId { get; set; }
        public int? CreatedAt { get; set; }
        public string? CompletionWindow { get; set; }
        public int? ExpiresAt { get; set; }
        public int? StartedAt { get; set; }
        public int? CompletedAt { get; set; }
        public int? FailedAt { get; set; }
        public int? ExpiredAt { get; set; }
        public int? CancelledAt { get; set; }
        public RequestCounts? RequestCounts { get; set; }
    }
}
